### PR TITLE
[Gutenberg] Implements a call from gutenberg editor to native image fullscreen preview

### PR DESCRIPTION
**Pull request by cameronvoell**
### Fixes [gb-mobile/1286](https://github.com/wordpress-mobile/gutenberg-mobile/issues/1286) 

`gutenberg-mobile` PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/1597
`gutenberg` PR: https://github.com/WordPress/gutenberg/pull/18618

### More Info (Description, Steps)
See `gutenberg-mobile` PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/1597

ios-fullscreen-video

Update release notes:

- [X] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift [hunk 1]
```diff
@@ -542,6 +542,13 @@ extension GutenbergViewController: GutenbergBridgeDelegate {
             DDLogError(message)
         }
     }
+
```
[the remark this answers is not in the document]
Comment (cameronvoell) [reply WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift]: Fixed ba7000d54c8b3aa06fe465d423cf3cd9b9814e0c

File: WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift [hunk 2]
```diff
@@ -542,6 +542,13 @@ extension GutenbergViewController: GutenbergBridgeDelegate {
...
+    func gutenbergDidRequestFullscreenImage(with mediaUrl: URL) {
```
Comment (cameronvoell) [inline WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift]: Inspired by https://github.com/wordpress-mobile/WordPress-iOS/blob/32314106bc4a7f6bf95768cac4fc12cac1c89d16/WordPress/Classes/ViewRelated/Media/MediaItemViewController.swift#L189-L195

File: RELEASE-NOTES.txt [hunk 3]
```diff
@@ -1,3 +1,7 @@
+13.8
+-----
+* Block editor: Images clickable for fullscreen preview
```
Comment (etoledom) [inline RELEASE-NOTES.txt]: Small detail, `clickable` might not be the best word?
What about something like:
```suggestion
* Block editor: Images from Image Block can now be previewed full screen by tapping on them.
```

Comment (cameronvoell) [reply RELEASE-NOTES.txt]: That does sound more clear. Updated, thanks!

**Comment by cameronvoell** [comment]
> In this way, all the animation happens over the AztecNavigationController (used to present Gutenberg), and it looks good :)

Nice catch, and thanks for explaining the fix. Implemented here: 08988b9ad4c494080b9fb5010de9d04857c0343f

See video of the better transition now:
video-better-transition

**Pull request merged**